index.html

```diff
@@ -182,6 +183,8 @@ <h2>Glossary</h2>
 
 <p><dfn id="def_code_point" class="lint-ignore">Code point</dfn>. A code point value represents the position of a character in a coded character set. For example, the code point for the letter <span class="qchar">á</span> in the Unicode coded character set is 225 in decimal, or 0xE1 in hexadecimal notation. Hexadecimal notation is commonly used for referring to code points. See also <a class="termref" href="#def_unicode_code_point">Unicode code point</a>.</p>
 
+<p><dfn data-lt="code unit|code units" id="def_code_unit" class="lint-ignore">Code unit</dfn>. The units of data used by a <a>character encoding</a>to encode or serialize characters into a programming language or other serialized form (such as a file). Common code units are 8-, 16-, and 32-bits in size. For example, in C, a <code>char</code> is generally an 8-bit byte: each <code>char</code> is a 8-bit code unit. In Java or Javascript, a <code>char</code> is a 16-bit code unit.</p>
```

Comment:
made a conscious decision at one point not to include _code unit_ in the glossary, because frankly i find that it is not actually a concept that is needed much unless you are deeply into coding theory (which i think most of our readers will not be), and that for others having to factor in this terminology just tends to make things more confusing.  Yes, the terms exist, but in what way does it help to define them here? The big exception, of course, is that code units become important when talking about older uses of JavaScript for supplementary characters, and perhaps we could adapt the definition to focus on that?

Reply:
It is difficult to talk about encodings without being drawn into the code unit/code point distinction. Notice that [Infra](https://infra.spec.whatwg.org/#strings) defines and uses code units as a term (where they mean JS chars).

We could focus the explanation on JavaScript (and DOM), since that's the most interesting code unit on the Web.